microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -552,8 +531,21 @@ To see the Object Properties viewer in the **Sources** tool:
 
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 109.
-*  This checkbox is present in Microsoft Edge Stable 107.
+*  This checkbox is present in Microsoft Edge Canary 111.
+*  This checkbox is present in Microsoft Edge Stable 108.
+
+
+<!-- ====================================================================== -->
+## Disable the deprecated 'Color format' setting (requires reloading DevTools)
+
+_new as of Microsoft Edge 110_
+
+Disables the deprecated **Color format**<!-- string not found by FTS --> setting.
+<!-- todo: where in UI? -->
```

Comment:
@captainbrosset - any clarification?

Reply:
This is related to the Color format preference (pictured below). This preference used to be available for users to choose what color format they preferred to see in the Styles pane by default. So, for example, if the CSS code specified `color: red;` but the user had set the `hex` color format as their preference, then they would see `color: #f00;` in the Styles pane instead.

Recently, CSS has gained new color formats that can be used to describe colors in color spaces that weren't available before.
This auto-switch of the color format can't work anymore, because if the source code contains a color in a new color space, then it isn't always possible to convert this color in one of the older color formats because the color spaces simply don't match 1 to 1.

It seems like this experiment was added just as a "grace" period. It's on by default, and its effect is to grey out the color format dropdown in the Preferences pane. After a while, the experiment will disappear, and the color format preference will disappear too.
For now, it's still there, for a short while, so people can still disable the experiment and regain control of the color format preference.

![image](https://user-images.githubusercontent.com/1152698/211495284-c752f421-93e4-4fa9-ae1e-964a7cdeb57a.png)

Reply:
added text, probably not screenshot.